include/common.h

```diff
@@ -50,6 +51,36 @@ bool getSdfParam(sdf::ElementPtr sdf, const std::string& name, T& param, const T
   return false;
 }
 
+template <typename T>
```

Comment:
Also, why is this still here while there's https://github.com/PX4/sitl_gazebo/pull/167?

Reply:
Yes, it is https://github.com/PX4/sitl_gazebo/pull/167. I included it because I wanted fully working branch https://github.com/acsl-mipt/sitl_gazebo/tree/multiple-sitl-pr